src/jit_compiler.hpp

```diff
@@ -28,10 +28,48 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #pragma once
 
-#if defined(_M_X64) || defined(__x86_64__)
+#include "common.hpp"
+
+namespace randomx {
+
+	struct CodeBuffer {
```

Comment:
If this struct is only used by risc-v compiler, maybe it should be in `jit_compiler_rv64.hpp`

Reply:
This PR contains some seemingly unused code that will be used by #274. That includes this struct (will be used by the X86 compiler) and some assembly code in `jit_compiler_rv64_static.S` (e.g. the soft AES code).